src/App.sass

```diff
@@ -2,6 +2,7 @@
 
 // Import a Google Font
 @import url('https://fonts.googleapis.com/css2?family=Poppins:400,700');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap');
```

Comment:
Just a quick note on coding style that isn't of much consequence: you can actually concatenate these imports into a single line.

`@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Roboto+Mono:wght@400;700&display=swap');`